xlators/mount/fuse/src/fuse-helpers.c

```diff
@@ -143,7 +143,6 @@ get_fuse_state (xlator_t *this, fuse_in_header_t *finh)
 }
 
 
-#define FUSE_MAX_AUX_GROUPS 32 /* We can get only up to 32 aux groups from /proc */
```

Comment:
I guess my thinking here is that it is Linux specifically which is limited to 32 groups. Why does removing this particular limitation imply also checking for Linux. Shouldn't the default be much higher?